packages/utilities-deprecated/projects/utilities/CHANGELOG.md

```diff
@@ -9,6 +9,694 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+  Furthermore, the functions accept only the predefined breakpoints from `base` to `xxl` and no custom breakpoints
+  anymore.
+
+```diff
+- import { breakpoint, mediaQuery } from '@porsche-design-system/utilities';
++ import { breakpoint, getMediaQueryMin, getMediaQueryMax, getMediaQueryMinMax } from '@porsche-design-system/components-{js|angular|react|vue}/styles';
+import styled from 'styled-components';
+
+const Component1 = styled.div({
+  color: 'royalblue',
+
+  // up to predefined breakpoint xs apply color black
++ [getMediaQueryMax(breakpoint.xs)]: {
+    color: 'black'
+  }
+
+  // from predefined breakpoint xs to m apply color aqua
+- [mediaQuery(breakpoint.xs, breakpoint.m)]: {
++ [getMediaQueryMinMax(breakpoint.xs, breakpoint.m)]: {
+    color: 'aqua'
+  },
+
+  // from predefined breakpoint m apply color deeppink
+- [mediaQuery(breakpoint.m)]: {
++ [getMediaQueryMin(breakpoint.m)]: {
+    color: 'deeppink'
+  }
+}`;
+```
+
+- `titleLarge` got renamed to `displayLargeStyle`.
+
+```diff
+- import { titleLarge } from '@porsche-design-system/utilities';
++ import { displayMediumStyle, displayLargeStyle } from '@porsche-design-system/components-{js|angular|react|vue}/styles';
+import styled from 'styled-components';
+
+const Component1 = styled.h1`
+- ${titleLarge}
++ ${displayLargeStyle}
+`;
+
+const component2 = styled.h2`
++ ${displayMediumStyle}
+`;
+```
+
+- `headline{1|2|3|4|5}` got renamed to `heading{Small|Medium|Large|XLarge|XXLarge|XXXLarge}Style`.
+
+```diff
+- import { headline } from '@porsche-design-system/utilities';
++ import { headingXXLargeStyle } from '@porsche-design-system/components-{js|angular|react|vue}/styles';
+import styled from 'styled-components';
+
+const Component1 = styled.h1`
+- ${headline['1']}
++ ${headingXXLargeStyle}
+`;
+```
+
+- `text` got renamed to `text{XSmall|Small|Medium|Large|XLarge}Style`.
+
+```diff
+- import { text } from '@porsche-design-system/utilities';
++ import { textSmallStyle } from '@porsche-design-system/components-{js|angular|react|vue}/styles';
+import styled from 'styled-components';
+
+const Component1 = styled.p`
+- ${text.small}
++ ${textSmallStyle}
+`;
+```
+
+**Colors**
+
+As of v3 of the Porsche Design System the color theme got reworked completely to achieve a monochrome look. To get an
+overview of the new colors have a look [here](https://designsystem.porsche.com/latest/styles/theme).
+
+- `color.lightTheme` and `color.darkTheme` got renamed to `theme.light` and `theme.dark`. In addition, you can also
+  single import a theme, e.g. if you only need `themeLight`.
+
+```diff
+- import { color } from '@porsche-design-system/utilities';;
++ import { theme, themeLight, themeDark } from '@porsche-design-system/components-{js|angular|react|vue}/styles';
+```
+
+Since the styles for `themeLight` and `themeDark` are synchronized, the name changes mentioned below affect both themes,
```

Comment:
This sentence belongs to the point "`brand` got renamed to `primary`", so I would move it there.

Because, when reading, I tried to apply this sentence to the point "`color.lightTheme` and `color.darkTheme` got renamed..." and got confused

Reply:
This belongs to all the below changes as there was `color.lightTheme`  and `color.darkTheme` which have the same keys. So it is the correct position